css/styles.css

```diff
@@ -65,6 +65,25 @@ body {
   list-style: none;
 }
 
+/* flex containers */
+.flex-nav li {
+  flex: 1;
+}
+
+.flex-nav .social {
+  flex: 1;
+}
+
+.cards {
+  display: flex;
+  flex-direction: column;
+  padding: 1rem;
+  gap: 5px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  background-color: #fff;
+}
```

Comment:
- [ ] Please as the last reviewer said the form section is not a card please change it and make sure it looks like the design![image](https://user-images.githubusercontent.com/74744787/186268623-8f658db3-4fa9-43ae-9434-c4b5f212ed2f.png)

Reply:
Sorry, my bad. A bit stressed. I didn't pay enough attention to the illustration